sqs/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/sinks/SqsSink.scala

```diff
@@ -307,7 +332,7 @@ object SqsSink {
   }
 
   /** Create an aws credentials provider through env variables and iam. */
-  private def getProvider(awsConfig: AWSConfig): Either[Throwable, AWSCredentialsProvider] = {
+  private def getProvider(awsConfig: SqsSinkConfig.AWSConfig): Either[Throwable, AWSCredentialsProvider] = {
```

Comment:
I know this is unrelated to your PR, but I really dislike this `getProvider` method.  At Snowplow we always use the default chain, and I cannot think of any situation when we would use any other method.  I would vote to remove this unhelpful functionality as part of the new collector release.